src/vs/workbench/contrib/chat/common/annotations.ts

```diff
@@ -17,6 +17,9 @@ export function annotateSpecialMarkdownContent(response: Iterable<IChatProgressR
 
 	const result: IChatProgressRenderableResponseContent[] = [];
 	for (const item of response) {
+		if (item.kind === 'thinking' && !item.value) {
```

Comment:
I thought we weren't sending empty thinking parts anymore? Is this still an issue?